### ISSUE TITLE
NVDA dows not announce content in table cells in OneNote

### ISSUE THREAD

**Issue by Adriani90** [nvaccess/nvda#8302]
### Steps to reproduce:
1. Press Windows+n to open a new note in OneNote
2. Press alt to open the ribbon bar and go to insert. Then, insert a table in that document
3. Write something in the first and the second cell.
4. Use ctrl+arrow keys to navigate through the table

### Expected behavior:
NVDA should report the content of a cell along with cell coordinates, like in MS Word.

### Actual behavior:
NVDA only reports cell coordinates, but not the content.

### System configuration:
NVDA version:
NVDA 2018.1.1

NVDA Installed or portable:
Installed

Windows version:
Windows 10.0.17134

Name and version of other software in use when reproducing the issue:
OneNote 2016

### Other questions:

Does the issue still occur after restarting your PC?
yes

Have you tried any other versions of NVDA?
no
